apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTestDatabase.cs

```diff
@@ -12,161 +12,55 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using Google.Api.Gax;
-using Google.Api.Gax.ResourceNames;
 using Google.Cloud.ClientTesting;
-using Google.Cloud.Spanner.Admin.Instance.V1;
-using Google.Cloud.Spanner.Common.V1;
-using Google.Cloud.Spanner.V1.Internal.Logging;
-using Grpc.Core;
+using Google.Cloud.Spanner.Admin.Database.V1;
 using System;
 
-namespace Google.Cloud.Spanner.Data.CommonTesting
+namespace Google.Cloud.Spanner.Data.CommonTesting;
```

Comment:
I have used the file scoped namespace here, since the entire file was getting modified. Happy to revert, if it is difficult to review this way.